### DISCUSSION TITLE
SDL being funny

### DISCUSSION THREAD

**Discussion by RaphaelIT7** [Source-Authors/Obsoletium#112] [Q&A] [answered]
Had this now for a while that SDL would not want to compile, I've always nuked the `SDL_PrintFloat` function like this to get it working:
```cpp
static size_t SDL_PrintFloat(char *text, size_t maxlen, SDL_FormatInfo *info, double arg, SDL_bool g)
{
	return 0;
}
```
Since else it throws this at me, complaining that it can't resolve `__ftoul2_legacy`
(Compiler is German :/)`SDL_string.obj : error LNK2019: Verweis auf nicht aufgelöstes externes Symbol "__ftoul2_legacy" in Funktion "_SDL_PrintFloat".`

Now I was wondering, is there a way to solve it without having to always nuke the function local?

**Comment by dimhotepus** [comment, marked as answer]
Well, I'm building SDL fine using one of ```create_<game name>_<arch>.bat``` scripts.

Build failure is due to 
https://github.com/actions/runner-images/issues/10980
https://github.com/libsdl-org/SDL/issues/11487

```
D:\a\Obsoletium\Obsoletium\thirdparty\SDL\src\joystick\gdk\SDL_gameinputjoystick.c(29,10): error C1083: Cannot open include file: 'gameinput.h': No such file or directory [D:\a\Obsoletium\Obsoletium\thirdparty\SDL\out\SDL3-shared.vcxproj]
  (compiling source file '../src/joystick/gdk/SDL_gameinputjoystick.c')  
```

Replaced CMake generator for SDL from MSVC to Ninja as suggested and SDL now builds.
